content/applications/inventory_and_mrp/manufacturing/management/quality_control.rst

```diff
@@ -0,0 +1,80 @@
+================================
+Configure quality control checks
+================================
+
+Quality control is an essential part of the manufacturing process. Odoo's Quality module can be
+used to control the quality of products before they are registered into stock, during picking
+operations, and when leaving the warehouse in a delivery order. This is accomplished by creating
+*Quality Control Points* that will automatically trigger quality checks at specific points during
+production.
+
+Create and configure quality control points
+===========================================
+
+To create a quality control point, go to :menuselection:`Quality Control --> Control Points -->
+Create`. From here, configure the new control point based on the desired quality check workflow.
+
+.. image:: quality_control/quality-control-form.png
+   :align: center
+   :alt: Configure a quality control point in Odoo.
```

Comment:
I see what you did here and in the sections that follow, by briefly pointing to the UI and then describing it, as needed, in sections below. 

This is the right idea, however we want to do a deeper walk-through right in the beginning instead of forcing the user to scroll up and down the page to find more information. This section, therefore, is way too light/lean.

Instead of a single sentence overview + screenshot, would do a bulleted list here where each UI item has it's _own_ single sentence overview.

For the screenshot, the generic one isn't useful since none of the fields are filled in. A better screenshot imo would be to demonstrate a filled in use case, like this: https://i.imgur.com/YeEw88b.png

Notice too, the `note` I added at the end which adds more context on deeper fields. This is a way to address the finer points of configuration without being overly exhaustive in one section. This also opens up room for deeper context in the following sections.

Again, right idea, would just kick up the depth overall and introduce that depth much sooner in the doc.

```suggestion
To create a new quality control point, go to :menuselection:`Quality Control --> Control Points -->
Create`. Then, fill in the primary control point form fields, as follows:

- :guilabel:`Title`: give the quality control point a simple, but informative title so production
  floor and quality check teams can understand it easily
- :guilabel:`Products`: indicate which product(s) should pass through the specific quality control
  point
- :guilabel:`Operations`: determine which operations team(s) should perform the quality control
  check, e.g. `Manufacturing`, `Receipts`, `Delivery Orders`, etc.
- :guilabel:`Work Order Operation`: using the drop-down menu, indicate which type of operation the
  quality control point applies to, during: :guilabel:`Manual Assembly`, :guilabel:`Packing`,
  :guilabel:`Assembly`, :guilabel:`Testing` or :guilabel:`Long time assembly`.
- :guilabel:`Company`: add the company that will implement the quality control point. Usually this
  will be the company that owns the Odoo database, however, a multi-company or vendor profile can
  go here as well for cases where there are multiple manufacturing or engineering locations

.. image:: quality_control/quality-control-point-form.png
   :align: center
   :alt: An example of a completed quality control point form, however there are many variations to
   choose from in Odoo Quality, depending on the manufacturing workflow.

In the right column of the form, continue to fill in the remaining fields, as follows:

- :guilabel:`Control Type`: using the drop-down menu, choose from :guilabel:`All Operations`,
  :guilabel:`Randomly` or :guilabel:`Periodically` to determine how often the control point should
  executed
- :guilabel:`Type`: using the drop-down menu, choose the control point type, from
  :guilabel:`Instructions`, :guilabel:`Take a Picture`, :guilabel:`Registered Consumed Materials`,
  :guilabel:`Print label`, :guilabel:`Pass - Fail`, or :guilabel:`Measure`
- :guilabel:`Team`: decide which quality team should receive the results of the quality control
  point check.
- :guilabel:`Responsible`: add a point person to manage the status and evolution of the quality
  control point over time
- :guilabel:`Worksheet`: decide what should happen next in the assembly instructions when the
  quality control point meets a desired outcome, either to :guilabel:`Scroll to specific page` or
  :guilabel:`Do not update page`

.. note::
   Some form fields, such as :guilabel:`Type` will open up more fields to specify further control
   point details. Choosing :guilabel:`Measure`, for example, adds deeper control point
   specifications to indicate what the measurement :guilabel:`Norm` should be, as well as minimum
   and maximum :guilabel:`Tolerance` levels that are still within satisfaction above and below the
   :guilabel:`Norm`.
```